src/shared/bg_public.h

```diff
@@ -151,6 +151,14 @@ enum team_t
   NUM_TEAMS
 };
 
```

Comment:
For stuff that is only used in one file, I'd rather see it defined next to the point of use, as it is now.

What exactly is your goal with this commit? I mean, in what aspect should the code be better than it was before?

Reply:
The point is to prepare a more uniform way to configure a whole team/species characteristics. I think it would be better to move stuff in an arrray of struct, really, and I'd do it without problems, but I wanted to know where and how it should be done to follow unv's style.

Reply:
IMO this centralized location only makes refactoring more difficult: it's impossible to tell what the fields are used for, so you have to jump back and forth between the global arrays and the point of use. It's more efficient when the stuff is defined at the point of use.

Let's say you're making a mod where aliens are replaced with kittens, then what you can do is rename `TEAM_ALIENS` to `TEAM_KITTENS`. Then you will get a compile error for each place `TEAM_ALIENS` was mentioned, which helps you to identify all the things that needed to be changed. The refactoring with `G_IsPlayableTeam` and team iterators will help to eliminate false positives for this method.